### ISSUE TITLE
Permanent Authorizations clearing

### ISSUE THREAD

**Issue by ghost** [Tonius/E-Mobile#12]
Everytime someone logs off my server, any permanent authorizations they have with other players are cleared. Any info on this?

**Comment by K4N0** [comment]
I agree we really could use a permanent authorization and possibly a way to see who is on the authorization list. Maybe a gui button to clear all authorized players.

**Comment by dries007** [comment]
I know how to fix this, you store the information in NBT data of the player correct?
I think you forgot to store it under the persistent object forge uses. (see EntityPlayer's PERSISTED_NBT_TAG)
You want me to PR in a fix?
